sdk/scheduler/src/main/java/com/mesosphere/sdk/http/PlansResource.java

```diff
@@ -421,4 +419,8 @@ private static JSONObject getCommandResult(String command) {
                 "message",
                 String.format("Received cmd: %s", command)));
     }
+
+    static JSONObject getPlanCommandResult(String cmd, String plan, String phase, String step) {
```

Comment:
This could be `private` but don't block on this

Reply:
oh yeah, thanks.  Artifact of earlier version.